programs/nodeos/main.cpp

```diff
@@ -94,6 +95,7 @@ int main(int argc, char** argv)
    try {
       app().set_version(eosio::nodeos::config::version);
       app().register_plugin<history_plugin>();
+      app().register_plugin<state_history_plugin>();
```

Comment:
This is not needed because of line 32 of state_history_plugin.cpp and line 54 of CMakeLists.txt.